### ISSUE TITLE
Is JavaScript a case-sensitive language?

### ISSUE THREAD

**Issue by bofeiw** [shamsmosowi/git-bot#25]

**Comment by shamsmosowi** [comment]
Yes, JavaScript is a case-sensitive language. This means that variable and function names must always be written with the same capitalization that you used when you defined them. 

For example, if you define a variable called “myVariable”, if you try to access the variable using “myvariable”, the code will not work, because the capitalization for the second instance of the variable is not the same. 

It is important to note that case-sensitivity applies not only to variable names, but also to keywords and functions. 

I hope this helps!
